.swiftformat

```diff
@@ -82,4 +82,5 @@
 --disable andOperator,opaqueGenericParameters,preferKeyPath,redundantReturn,strongOutlets,trailingCommas,unusedArguments,wrapMultilineStatementBraces
 --enable blankLineAfterImports,blankLinesBetweenImports
 --exclude DerivedData,Derived,Tuist,Project.swift,Build,.build
+--header /*\n Infomaniak Mail - iOS App\n Copyright (C) {created.year} Infomaniak Network SA\n\n This program is free software: you can redistribute it and/or modify\n it under the terms of the GNU General Public License as published by\n the Free Software Foundation, either version 3 of the License, or\n (at your option) any later version.\n\n This program is distributed in the hope that it will be useful,\n but WITHOUT ANY WARRANTY; without even the implied warranty of\n MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the\n GNU General Public License for more details.\n\n You should have received a copy of the GNU General Public License\n along with this program. If not, see <http://www.gnu.org/licenses/>.\n */
```

Comment:
So we have twice the GPLV3 header in two places. Could we make one depend on the other ?
Maybe point to the existing file is swiftlint has support for it.

Reply:
This is not possible for now but I will keep an eye on this